docs/docs/guides/05-type-system/03-cheatsheet.mdx

```diff
@@ -0,0 +1,255 @@
+---
+title: CheatSheet
+tags:
+  - typescript
+  - type system
+  - type
+  - cheatsheet
+toc_max_heading_level: 3
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+import Admonition from '@theme/Admonition'
+
+### Common Use Cases
+
+---
+
+#### Function that Returns Any Entity
+
+```typescript
+import type { Data } from '@strapi/types';
+
+function find(): Data.Entity {}
```

Comment:
Can you give a code example with a return to see when this might be used?

Reply:
Not sure I understand completely what you're referring too? Do you want an actual body definition for the function?

Reply:
Yeah, like a use case showing what you would select in order to want to return a generic Data.Entity. Not important though, just a thought.